docs/decisions/0003-static-builds-to-client-side-react.rst

```diff
@@ -0,0 +1,55 @@
+============================================================
+3. Move from static builds (Gatsby) to client-side React app
+============================================================
+
+******
+Status
+******
+
+Accepted
+
+*******
+Context
+*******
+
+Currently, the Enterprise and Programs learner portals are created using Gatsby, a static site generator based on React. Gatsby works by sourcing data (e.g., from external APIs) and creates static web pages (i.e., HTML, CSS, JavaScript) during the build process. This contrasts from a standard React application (or "microfrontend") where data is fetched client-side, or after the app has already been loaded in the browser. The Enterprise and Programs learner portals both utilize a custom Gatsby plugin to fetch the necessary data about learner portal configurations from the ``portal-designer`` service (i.e., ``gatsby-source-portal-designer``).
+
+Statically-built sites with Gatsby offer the following benefits over a client-side React app:
+
+* **SEO.** Web crawlers can more easily parse the HTML source of a static site to understand what a given page is about, as opposed to a client-side app where the HTML is created via JavaScript on the fly.
```

Comment:
I'd suggest talking about the benefits specific to the learner portal and removing `SEO` as I imagine we're not interested in the SEO of a portal internal to a specific enterprise

Reply:
Ah just saw you cover that down below. Ignore me then, just trying to shorten the doc